src/fragments/lib/auth/native_common/signin_next_steps/common.mdx

```diff
@@ -43,6 +43,96 @@ import iosConfirmSignInTotpMFA from '/src/fragments/lib/auth/ios/signin_next_ste
 
 <Fragments fragments={{ swift: iosConfirmSignInTotpMFA }} />
 
+<InlineFilter filters={['android']}>
+
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+Amplify.Auth.confirmSignIn(
+    "TOTP confirmation code",
+    result -> {
+        if (result.isSignedIn()) {
+            Log.i("AuthQuickstart", "Confirm signIn succeeded");
+        } else {
+            Log.i("AuthQuickstart", "Confirm sign in not complete. There might be additional steps: " + result.getNextStep());
+            // Switch on the next step to take appropriate actions.
+            // If `signInResult.isSignedIn` is true, the next step
+            // is 'done', and the user is now signed in.
+        }
+    },
+    error -> Log.e("AuthQuickstart", "Confirm sign in failed: " + error)
+);
+```
+</Block>
+
+<Block name="Kotlin - Callbacks">
+
+```kotlin
+Amplify.Auth.confirmSignIn(
+    "TOTP confirmation code",
+    { result ->
+        if (result.isSignedIn) {
+            Log.i("AuthQuickstart", "Confirm signIn succeeded")
+        } else {
+            Log.i(
+                "AuthQuickstart",
+                "Confirm sign in not complete. There might be additional steps: ${result.nextStep}"
+            )
+            // Switch on the next step to take appropriate actions.
+            // If `signInResult.isSignedIn` is true, the next step
+            // is 'done', and the user is now signed in.
+        }
+    },
+    { error -> Log.e("AuthQuickstart", "Confirm sign in failed: $error") }
+)
+```
+</Block>
+
+<Block name="Kotlin - Coroutines">
+
+```kotlin
+try {
+    val result = Amplify.Auth.confirmSignIn(
+        "confirmation code"
```

Comment:
nit:
```suggestion
        "TOTP confirmation code"
```
to match

Reply:
Thanks good catch - had copied this from the existing fragment 😄